### PR TITLE
Adding VM owner change and Revoke ability to roles and users

### PR DESCRIPTION
**Pull request by philippreston**
Hey 

Just with ongoing use of fifo - would be useful to have change owner available on the command line, as well as the ability to revoke permissions.

Thanks

Phil

### CODE REVIEW

File: doc/fifo-roles.1 [hunk 1]
```diff
@@ -40,7 +40,7 @@ the data base.
 roles create [\-p] NAME
 Creates a new role, if \-p is passed only the uuid is returned.
 .TP
-roles grant ROLE PERM...
+roles grant <uuid> ROLE PERM...
```
Comment (Licenser) [inline doc/fifo-roles.1]: the `<uuid>` is double here, `ROLE` is already a placeholder

File: doc/fifo-roles.1 [hunk 2]
```diff
@@ -49,6 +49,9 @@ is the uuid of the role and
 .B PERM
 is one or more permission elements, i.e. cloud vms create
 .TP
+roles revoke <uuid> ROLE PERM...
```
Comment (Licenser) [inline doc/fifo-roles.1]: see previous comment

**Comment by philippreston** [comment]
Sorry misread - fixed in line with comments and committed.

**Comment by Licenser** [comment]
Awesome thanks mate!

**Pull request merged**